### PR TITLE
feat: clearer platform support in settings

### PR DESCRIPTION
**Pull request by pauldambra**
We have multiple controls and config, and they're not all available on all platforms.

In a perfect world they would be, in a future world they will be... but for now, let's try and make it clearer

### CODE REVIEW

**Comment by pauldambra** [comment]
@ioannisj @annikaschmid @marandaneto very quick first pass of how we could make platform support more clear
gives us a place we could have pop-ups for what version support was introduced or to link to the feature request for the support for folk to vote on

**Comment by marandaneto** [comment]
> @ioannisj @annikaschmid @marandaneto very quick first pass of how we could make platform support more clear gives us a place we could have pop-ups for what version support was introduced or to link to the feature request for the support for folk to vote on

That's cool!

**Comment by pauldambra** [comment]
> Should we check Flutter as well for Canvas capture?

Ah, interesting... yep. I think this needs to be a note as well since it reads like it won't work _without_ canvas enabled

**Comment by marandaneto** [comment]
> canvaskit renderer

yes, thats correct, but only for the canvaskit renderer (which isn't the default).

**Review by ioannisj — APPROVED** [review]
💪

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 1]
```diff
@@ -8,18 +18,92 @@ import { PropertySelect } from 'lib/components/PropertySelect/PropertySelect'
...
+}
+
 function LogCaptureSettings(): JSX.Element {
     const { updateCurrentTeam } = useActions(teamLogic)
     const { currentTeam } = useValues(teamLogic)
 
     return (
         <div>
             <h3>Log capture</h3>
+            <SupportedPlatforms android={true} ios={true} flutter={true} web={true} reactNative={true} />
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:106]: ios and flutter should be false

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 2]
```diff
@@ -111,6 +208,7 @@ export function NetworkCaptureSettings(): JSX.Element {
 
     return (
         <>
+            <SupportedPlatforms android={true} ios={true} flutter={false} web={true} reactNative={false} />
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:211]: RN should be true here

**Pull request merged**